cmd/gen/makefile/template/template.go

```diff
@@ -0,0 +1,67 @@
+package template
+
+//TemplateConfig inherits all placeholder which should be overwritten.
+type TemplateConfig struct {
+	Application string
+}
+
+var Makefile = `APPLICATION   ?= {{.Application}}
+VERSION       ?= $(shell git describe --tags --always --dirty)
+SOURCES       = $(shell find . -name '*.go')
+GOPKGS        = $(shell go list ./...)
+TAG           ?= $(VERSION)
+BUILD_FLAGS   ?= -v
+
+default: build
+
+.PHONY: build
+## build: builds a local binary
+build: clean $(SOURCES)
+	CGO_ENABLED=0 go build -o ${APPLICATION} $(BUILD_FLAGS) .
+
+.PHONY: build-linux
+## build-linux: builds binary for linux/amd64
+build-linux: clean $(SOURCES)
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build $(BUILD_FLAGS) .
+
+.PHONY: build-darwin
+## build-darwin: builds binary for darwin/amd64
+build-darwin: clean $(SOURCES)
+	GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 go build $(BUILD_FLAGS) .
+
+.PHONY: install
+## install: install the application
+install: $(SOURCES)
+	go install $(BUILD_FLAGS) .
+
+.PHONY: run
+## run: runs go run main.go
+run: $(SOURCES)
+	go run -race .
+
+.PHONY: clean
+## clean: cleans the binary
+clean:
+	go clean
+
+.PHONY: lint
+## lint: runs golangci-lint
+lint:
+	golangci-lint run --timeout=15m ./...
+
+.PHONY: test
+## test: runs go test with default values
+test:
+	go test -v -race -cover $(GOPKGS)
```

Comment:
I'd drop `-v` here as well. We've been in a place where we had so many packages that it was difficult to find what was failing when tests were red.